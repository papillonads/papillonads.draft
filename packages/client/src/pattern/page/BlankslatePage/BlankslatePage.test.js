import React from 'react'
import blankslatePageDefault from './BlankslatePage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<BlankslatePage />', () => {
  describe('Snaphot', () => {
    test('must match blankslatePageDefault', () => {
      expect(global.renderToJSON(blankslatePageDefault)).toMatchSnapshot()
    })
  })
})
