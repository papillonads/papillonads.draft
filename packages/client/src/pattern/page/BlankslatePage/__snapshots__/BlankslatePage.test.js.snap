// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlankslatePage /> Snaphot must match blankslatePageDefault 1`] = `<div />`;
